app/src/main/res/drawable/ic_star_full.xml

```diff
@@ -0,0 +1,8 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
:thinking: Commit message: Why `Replace Reminder Icons`? How about `Replace icons to un/favor a session.`?

Side note: you do not have to capitalize most of the English words in your commit messages. Also have read [here](https://cbea.ms/git-commit/).